internal/training/grpo_engine/bench_io.c

```
@@ -0,0 +1,116 @@
+        stats.bytes_read = (uint64_t)st.st_size;
+    }
+
+    double elapsed_s = (double)stats.read_ns / 1e9;
+    stats.throughput_gbps = elapsed_s > 0 ? ((double)stats.bytes_read / 1e9) / elapsed_s : 0;
+
+    stream_close(se);
+    free(logprobs);
+    return stats;
+}
+
+StreamComparison stream_benchmark_comparison(const char *gguf_path,
```

Cppcheck (2.21.0)

[style] The function 'stream_benchmark_comparison' is never used.

(unusedFunction)